Home.md

```diff
@@ -0,0 +1,56 @@
+![logo](https://github.com/culturegraph/metafacture-core/wiki/img/metafacture_small.png)
+
+# Metafacture Documentation
+
+Metafacture is a toolkit for processing semi-structured data with a focus on library metadata. It provides a versatile set of tools for reading, writing and transforming data. Metafacture can be used as a stand-alone application via CLI or as a Java library in other applications. There is also a playground where you can test workflows.
+
+The name Metafacture is a portmanteau of the words metadata and manufacture.
+
+Metafacture comprises three main parts: Framework, Flux and the Transformation-Module Fix. It can be extended with modules.
+
+__________________
+
+## Using Metafacture via playground or CLI
+
+> [!NOTE]
+> No JAVA-Code is necessary!!!
+
+While working with the playground or the command line you only need [Flux](#flux) and the transformation module [Fix](#fix).
+Have a look here for [Getting started](/Getting-Started.md).
+
+## Framework for JAVA-Integration/Development
+
+If you plan to use Metafacture as a Java library or if you wish to add commands to Flux. You should get familar with the [Framework](/Home.md#framework).
```

Comment:
```suggestion
```

Reply:
This was written before in the section describing using metafacture as a java library.

Reply:
not sure what you mean here?

Reply:
It is not written _before_ but _later_ in this section:
In this file `Home.md` you have two times the sentence 
see https://github.com/metafacture/metafacture-documentation/blob/a096b54186d6502cd8881426dc21ed5125490ff0/Home.md#framework-for-java-integrationdevelopment
and see https://github.com/metafacture/metafacture-documentation/blob/a096b54186d6502cd8881426dc21ed5125490ff0/Home.md#framework.

Reply:
-1 for now i would keep it. Lets have a look when it is productive